packages/frontend/core/src/modules/feature-flag/constant.ts

```diff
@@ -280,7 +280,7 @@ export const AFFINE_FLAGS = {
     description:
       'com.affine.settings.workspace.experimental-features.enable-code-block-html-preview.description',
     configurable: isCanaryBuild,
-    defaultState: true,
+    defaultState: false,
```

Comment:
```suggestion
    defaultState: isCanaryBuild,
```

Should be better if set it to `isCanaryBuild` for passing e2e tests?